docs/source/index.mdx

```diff
@@ -25,6 +24,8 @@ title: Apollo Server 4 (Alpha)
 *  **Universal compatibility** with any data source, any build tool, and any GraphQL client
 *  **Production readiness**, enabling you to confidently run your graph in production
 
+> Are you an integration author looking to build or help maintain an Apollo Server integration? Check out [Building Integrations](./integrations/building-integrations) for more details! 
```

Comment:
Oh, I meant more like, the line above that mentions some middleware could link to the integrations page. idk if this line needs to be part of what's a pretty small page.

Reply:
Makes sense to me! Thank you, I updated the above point to:

> * An add-on to your application's existing Node.js [middleware](./integrations/integrations-index) (such as [Express](./api/express-middleware), AWS Lambda, or Fastify)